src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -137,17 +146,9 @@ public Worker(WorkflowService.Iface service, String domain, String taskList, Wor
      * The reason for registration accepting workflow class, but not the workflow instance is
      * that workflows are stateful and a new instance is created for each workflow execution.
      * </p>
-     *
-     * @param workflowImplementationClasses
      */
     public void registerWorkflowImplementationTypes(Class<?>... workflowImplementationClasses) {
```

Comment:
can you also document that register is a SET operation - it will overwrite previously registered types (assuming I understand this API right)